nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1488,7 +1497,40 @@ private void checkFieldInitialization(ClassTree tree, VisitorState state) {
       // initialization block
       // anyways).
       errorBuilder.reportInitErrorOnField(
-          uninitSField, state, buildDescription(getTreesInstance(state).getTree(uninitSField)));
+          uninitSField,
+          state,
+          buildDescription(getTreesInstance(state).getTree(uninitSField)),
+          handler);
+    }
+    if (config.fixSerializationIsActive()) {
+      fixInitializationErrorsOnControlFlowPaths(state, errorFieldsForInitializer);
+    }
+  }
+
+  /**
+   * It only gets executed when {@code suggestEnabled} is activated in {@link
+   * com.uber.nullaway.fixserialization.FixSerializationConfig}. It creates a {@link
+   * com.uber.nullaway.fixserialization.out.SuggestedFixInfo} object for every class field that is
+   * not guaranteed to be {@code @Nonnull} at exit point.
+   *
+   * @param errorFieldsForInitializer Fields that are not initialized.
+   * @param state Visitor state.
+   */
+  private void fixInitializationErrorsOnControlFlowPaths(
```

Comment:
This method is not fixing errors.  Let's give it a name that more accurately reflects what it does.